PracticeTemplate/Controllers/Ex6/Ex6ViewController.swift

```diff
@@ -10,8 +10,50 @@ import UIKit
 
 class Ex6ViewController: BaseViewController {
     
+    // MARK: - IBoulet
+    @IBOutlet private weak var grayView: UIView!
+    @IBOutlet private weak var blueView: UIView!
+    @IBOutlet private weak var thumbView: UIView!
+    @IBOutlet private weak var thumbLabel: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupThumb()
+        setupLabel()
+    }
+    
+    // MARK: - Action
+    private func setupThumb() {
+        thumbView.layer.cornerRadius = thumbView.bounds.width / 2
+        thumbView.backgroundColor = .orange
+        thumbView.center = CGPoint(x: grayView.frame.midX, y: grayView.frame.midY)
+    }
+    private func setupLabel() {
+        thumbLabel.text = "%"
```

Comment:
Chổ này nếu set giá trị mặc định thì set ở XIB luôn cũng đc
```suggestion
        thumbLabel.text = "0%"
```